clients/python/agentic-sandbox-client/k8s_agent_sandbox/gke_extensions/snapshots/sandbox_with_snapshot_support.py

```
@@ -115,52 +126,66 @@ def _is_restored_from_snapshot(self, snapshot_uid: str) -> RestoreCheckResult:
+                logger.info(
+                    f"Sandbox '{self.sandbox_id}' is in the process of suspending (spec.operatingMode='{OPERATING_MODE_SUSPENDED}' but podIPs still present)."
+                )
             elif not is_spec_suspended and not pod_ips:
-                logger.info(f"Sandbox '{self.sandbox_id}' is in the process of resuming/starting (spec.operatingMode='{spec_operating_mode}' but no podIPs assigned).")
-                
+                logger.info(
+                    f"Sandbox '{self.sandbox_id}' is in the process of resuming/starting (spec.operatingMode='{spec_operating_mode}' but no podIPs assigned)."
+                )
+
             return is_spec_suspended
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)